cisa-advisory/info.json

```diff
@@ -0,0 +1,407 @@
+{
+    "name": "cisa-advisory",
```

Comment:
There is a mismatch between the name mentioned here and the repo name plz check on it

Reply:
Repo name would be changed after merging PR